mypy/fastparse.py

```diff
@@ -472,10 +472,15 @@ def visit_AugAssign(self, n: ast35.AugAssign) -> OperatorAssignmentStmt:
     # For(expr target, expr iter, stmt* body, stmt* orelse, string? type_comment)
     @with_line
     def visit_For(self, n: ast35.For) -> ForStmt:
+        if n.type_comment is not None:
+            target_type = parse_type_comment(n.type_comment, n.lineno, self.errors)
```

Comment:
This block refers to `index` as `target`, so I was just consistent. Having `target` and `target_type` become `index` and `index_type` is clearer than having `target` and `index_type` become `index` and `index_type`.